test-runner-frontend/src/Components/TestRunsTable.js

```diff
@@ -120,15 +127,28 @@ export default function TestRunsTable({ onCheckboxChange, isCheckBoxActive, test
                     } else if (column.id !== 'id') {
                       cell = value;
                     } else {
-                      cell = (
-                        <IdCell
-                          data-testid="checkboxTest"
-                          value={value}
-                          hasCheckbox={isCheckBoxActive}
-                          onChange={() => onChange(row)}
-                          status={row.status}
-                        />
-                      );
+                      cell =
+                        row.status === 'CREATED' || row.status === 'RUNNING' ? (
+                          <IdCell
+                            data-testid="checkboxTest"
+                            value={value}
+                            hasCheckbox={isCheckBoxActive}
+                            onChange={() => onChange(row)}
+                            status={row.status}
+                            textDecoration={'none'}
+                          />
+                        ) : (
+                          <Link to="/report" state={{ from: row }} style={{ textDecoration: 'none', color: 'black' }}>
```

Comment:
if we wrap the whole component in a link, what happens with the checkbox within `IdCell`?

Reply:
It will redirect on checkbox click. I should handle that on IdCell.